srcpkgs/firefox-esr/template

```diff
@@ -62,10 +53,10 @@ do_build() {
 	case "$XBPS_TARGET_MACHINE" in
 	*-musl)
 		echo "ac_add_options --disable-jemalloc" >>.mozconfig
-		echo "ac_add_options --enable-gold=no" >>.mozconfig
-		;;
-	arm*|aarch64*)
-		echo "ac_add_options --enable-gold=no" >>.mozconfig
+		echo "ac_add_options --disable-gold" >>.mozconfig
+		echo "ac_add_options --enable-release" >>.mozconfig
+		echo "ac_add_options --host=${XBPS_TRIPLET}" >>.mozconfig
+		echo "ac_add_options --target=${XBPS_TRIPLET}" >>.mozconfig
```

Comment:
This seems very wrong to me and most likely is the cause of the cross-compile woes.

Reply:
@Cogitri : thanks for your inputs.

these --host/--target directives are needed for musl build. 
In case of crossbuild, this directive is also used (few lines below), that makes it "working": 
```
echo "ac_add_options --target=$XBPS_CROSS_TRIPLET" >>.mozconfig
```